src/avfoundation.cs

```diff
@@ -10696,7 +10803,6 @@ interface AVCaptureDevice {
 		[return: NullAllowed]
 		NSData GetExtrinsicMatrix (AVCaptureDevice fromDevice, AVCaptureDevice toDevice);
 
-		[Unavailable (PlatformName.MacCatalyst)]
```

Comment:
Default for Catalyst?

Reply:
Catalyst was introduced for this member `globalToneMappingEnabled, if that's what you're indicating!